### PR TITLE
Make tests fail on deprecation notices

### PR DESCRIPTION
**Pull request by Ener-Getick**
Relates to #1197

### CODE REVIEW

**Comment by Ener-Getick** [comment]
It seems that there is no longer deprecation warnings with symfony 2.7.

**Comment by lsmith77** [comment]
uhm .. why is travis not running the tests?

**Comment by Ener-Getick** [comment]
Probably because of [this](https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/.travis.yml#L20 it was included in 07d02e5e1282475ba8d090868f7dbe406ae3c458). 

@stof do you know why travis is not running the tests for this PR ?

**Comment by Ener-Getick** [comment]
The build is run twice when I remove the ``branches`` section ...

File: .travis.yml [hunk 1]
```diff
@@ -17,11 +17,6 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-branches:
-  only:
-    - master
-    - /^\d+\.\d+$/
```
Comment (stof) [inline .travis.yml]: This is totally unrelated to the title of the PR, and it must not be removed

**Comment by stof** [comment]
@Ener-Getick PR builds are not triggered when the PR is conflicting. This may have been the issue

**Comment by Ener-Getick** [comment]
I removed the last commit and rebased but Travis is still not launched ...

**Comment by stof** [comment]
hmm, this makes me think that Travis broke something on their end

File: .travis.yml [hunk 2]
```diff
@@ -26,7 +22,7 @@ matrix:
   fast_finish: true
   include:
     - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER: weak
```
Comment (stof) [inline .travis.yml]: your Yaml is invalid (because of the ``:`` here). This is why there is no Travis build

Comment (Ener-Getick) [inline .travis.yml]: Right stupid error....

**Comment by Ener-Getick** [comment]
That's better thanks @stof
